src/proxy/NearCachedMapProxy.ts

```diff
@@ -194,7 +194,14 @@ export class NearCachedMapProxy<K, V> extends MapProxy<K, V> {
     }
 
     protected removeAllInternal(predicate: Predicate): Promise<void> {
-        this.nearCache.clear();
+        try {
+            this.nearCache.clear();
+        } catch (e) {
+            if (e instanceof SchemaNotReplicatedError) {
+                return this.registerSchema(e.schema, e.clazz).then(() => this.removeAllInternal(predicate));
```

Comment:
After you add a test to CompactPublicAPIsTest as I mentioned, coverage warnings for this method will also disappear

Reply:
not important, because you should remove this catch block.